swagger.json

```diff
@@ -3600,6 +3600,18 @@
                     "description": "Hours spent on duty or driving, rounded to two decimal places.",
                     "example": 5.40
                   },
+                  "onDutyMs": {
+                    "type": "number",
+                    "format": "int64",
+                    "description": "Duration in milliseconds that driver was on duty during the requested time range",
```

Comment:
super nit: period at the end of the sentence

Reply:
Addressed